packages/firestore-types/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@firebase/firestore-types",
-  "version": "0.1.0",
+  "version": "0.2.0",
```

Comment:
I'm surprised we'd need to bump the version number outside of the release process...  Hopefully @jshcrowthe can shed some light.